### PR TITLE
Feature: Add Syntax Highlighting

### PR DESCRIPTION
**Pull request by sghoweri**
1st pass adding syntax highlighting (powered by Prism) to the existing code block component.

### CODE REVIEW

File: components/pretty-code/src/map-children.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+// ported from https://github.com/rexxars/react-refractor/blob/master/src/mapChildren.js
+
+export function mapWithDepth(depth) {
+  return function mapChildrenWithDepth(child, i) {
+    // eslint-disable-next-line no-use-before-define
+    return mapChild(child, i, depth);
+  };
+}
+
+export function mapChild(child, i, depth) {
```
Comment (EvanLovely) [inline components/pretty-code/src/map-children.js]: Can we get some JSdoc comments on these two functions? If it's not possible or too hard, that's ok.

File: components/pretty-code/src/pretty-code.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+const js = require('refractor/lang/javascript');
+const json = require('refractor/lang/json');
+const jsx = require('refractor/lang/jsx');
+const markdown = require('refractor/lang/markdown');
+const markup = require('refractor/lang/markup');
+const markupTemplating = require('refractor/lang/markup-templating');
+const php = require('refractor/lang/php');
+const scss = require('refractor/lang/scss');
+const twig = require('refractor/lang/twig');
+const bash = require('refractor/lang/bash');
```
Comment (EvanLovely) [inline components/pretty-code/src/pretty-code.jsx]: Can we move all these `require` statements to `import`?

File: components/pretty-code/src/pretty-code.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+const refractor = require('refractor/core.js');
+
+// Load any languages you want to use from `refractor`
+const js = require('refractor/lang/javascript');
+const json = require('refractor/lang/json');
+const jsx = require('refractor/lang/jsx');
+const markdown = require('refractor/lang/markdown');
+const markup = require('refractor/lang/markup');
+const markupTemplating = require('refractor/lang/markup-templating');
+const php = require('refractor/lang/php');
```
Comment (EvanLovely) [inline components/pretty-code/src/pretty-code.jsx]: We can remove the `php` language.

File: components/pretty-code/src/pretty-code.styles.js [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+  word-spacing: normal;
+  word-break: normal;
+  word-wrap: break-word;
+
+  color: white;
+  background-color: hsl(0, 0%, 8%); /* #141414 */
+  text-shadow: 0 -0.1em 0.2em black;
+`;
+
+export const Tokens = styled.div`
```
Comment (EvanLovely) [inline components/pretty-code/src/pretty-code.styles.js]: Can we change `Tokens` to something like `CodeBlockTokens`? I don't want them confused with Design Tokens.

Comment (sghoweri) [reply components/pretty-code/src/pretty-code.styles.js]: Should be all set!

File: components/pretty-code/src/pretty-code.jsx (hunk 2 above)
Comment (sghoweri) [reply components/pretty-code/src/pretty-code.jsx]: All set!

File: components/pretty-code/src/map-children.js (hunk 1 above)
Comment (sghoweri) [reply components/pretty-code/src/map-children.js]: I made an attempt to get some JSDoc comments in for those two util functions, although I'll admit that the AST and VDOM-specific logic makes this a bit tricky. LMK!

Comment (EvanLovely) [reply components/pretty-code/src/map-children.js]: For tricky AST `any` works for me! 👍

**Comment by EvanLovely** [comment]
Released in [v0.20.0](https://github.com/basaltinc/bedrock/releases/tag/v0.20.0)

**Pull request merged**